pulsar-client-tools/src/main/java/org/apache/pulsar/admin/cli/CmdFunctions.java

```diff
@@ -661,10 +661,10 @@ void runCmd() throws Exception {
 
```

Comment:
as part of this pr, can you also get rid of all dependencies from pulsar-admin of any bookkeeper stuff?

Reply:
@srkukarni Thanks. removed un-used imports.

Reply:
@jiazhai I think there are still some references in pom.xml in pulsar-admin about bookkeeper dependencies. Maybe we can now remove them as well?